### PR TITLE
Feat: Change schedule meeting flow and redesign client thank-you page

### PR DESCRIPTION
**Pull request by cp-dharti-r**
## Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [x] Follow guidelines for creating branch name, adding commit messages or PR title
- [x] Add MR description if applicable
- [x] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [x] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [x] Remove unused or commented code if not required
- [x] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [x] In progress MR should be marked as draft
- [x] Make MR mark as ready before submitting it for review
- [x] Add images, videos, or gifs of UI
- [x] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

**Design should be tested on:**

- [ ] Inspector (all iPhones and android devices like samsung, motorola)
- [ ] Desktop (chrome, firefox, safari)
- [ ] Laptop / Macbook (chrome, firefox, safari)
- [ ] iPhone(safari)
- [ ] Available android devices (chrome, firefox)
- [ ] iPad(safari)
- [ ] Tablet (chrome, firefox)

## PR Description
- Redesign client thank-yoou page
- Change flow of schedule meeting from thank-you page to contact page

## Current pagespeed score for mobile and desktop
https://pagespeed.web.dev/analysis/https-dev-stack-canopas-com/68hluodmrz?form_factor=mobile

### CODE REVIEW

File: nuxt-frontend/config.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ export default Object.freeze({
   DESIGNRUSH_URL:
     "https://www.designrush.com/agency/website-design-development/in",
   BLOG_URL: "https://blog.canopas.com",
-  CALENDLY_IFRAME_URL: "https://calendly.com/jimmy822/30min",
+  CALENDLY_IFRAME_URL: "https://calendly.com/dharti-r-canopas/30min",
```
Comment (cp-sumi-k) [inline nuxt-frontend/config.js]: Was it intentional?

Comment (cp-dharti-r) [reply nuxt-frontend/config.js]: just refresh the changes

**Review by cp-sumi-k — DISMISSED** [review]

File: nuxt-frontend/stores/tags/index.js [hunk 2]
```diff
@@ -13,13 +13,14 @@ export const useTagListStore = defineStore("tag-list", {
...
   actions: {
-    async loadTagBlogs(slug) {
+    async loadTagBlogs(showDrafts, slug) {
       return new Promise((resolve, reject) => {
         this.isLoading = true;
         this.error = null;
 
-        let url =
-          config.API_BASE + "/api/posts/tags/" + slug + "?is_published=true";
+        let published = showDrafts ? "is_published=false" : "is_published=true";
```
Comment (cp-jimmy-s) [inline nuxt-frontend/stores/tags/index.js:21]: It can be `is_published=${showDrafts}`, if showDrafts is boolean

Comment (nidhi-canopas) [inline nuxt-frontend/stores/tags/index.js:21]: It can be is_published=${showDrafts}, if showDrafts is boolean.

**Review by nidhi-canopas — APPROVED** [review]

**Pull request merged**